layout/hp-header.tex

```diff
@@ -13,8 +13,8 @@
 
 \usepackage[bookmarks=true,unicode=true,pdfborder={0 0 0},
 	pdftitle={Harry Potter and the Methods of Rationality},
-	pdfauthor={Eliezer Yudkowsky}, breaklinks={true},
-	pdfkeywords={Harry Potter, rationality},pdfencoding=auto
+	pdfauthor={Yudkowsky, Eliezer}, breaklinks={true},
```

Comment:
I think I commented elsewhere that I don't think it's necessary (or a good idea) to reverse the names in `\pdfauthor`; do you have information to the contrary? (I presume you did it for a reason!)

Reply:
I reversed it, since in the ebook management tool calibri it seems to be the default to use "lastname, firstname"
your other comment as here: https://github.com/rrthomas/hpmor/pull/86#pullrequestreview-997147351

Reply:
OK, I respect Calibre (or rather Kovid Goyal!) but I'd like to have a bit more time to learn about this issue; could you possibly remove this change from this PR, as otherwise this one is fine to merge?

Reply:
done